cloud/jenkins/psmdb_operator_aks_latest.groovy

```diff
@@ -1,75 +1,95 @@
+aksLocation='westeurope'
```

Comment:
we renamed `GKEregion` to `region`, so why not rename `aksLocation` to `location`?

Reply:
ok